### PR TITLE
[Improve][e2e] Unified e2e IT for Redis

### PR DESCRIPTION
**Pull request by ashulin**
## Purpose of this pull request
- Unified e2e IT for Redis

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml [hunk 1]
```diff
@@ -18,30 +18,32 @@
...
 
     <dependencies>
         <dependency>
-            <groupId>org.apache.seatunnel</groupId>
-            <artifactId>connector-flink-e2e-base</artifactId>
-            <version>${project.version}</version>
-            <classifier>tests</classifier>
-            <type>test-jar</type>
+            <groupId>redis.clients</groupId>
+            <artifactId>jedis</artifactId>
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml]: remove?
already exists in `connector-redis`

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml [hunk 2]
```diff
@@ -18,30 +18,32 @@
...
         <!-- SeaTunnel connectors -->
         <dependency>
             <groupId>org.apache.seatunnel</groupId>
             <artifactId>connector-redis</artifactId>
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.seatunnel</groupId>
+            <artifactId>connector-fake</artifactId>
```
Comment (hailin0) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml]: unused?

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml (hunk 1 above)
Comment (ashulin) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml]: My fault, dependencies that are not provided don't need to be handled like this.

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml (hunk 2 above)
Comment (ashulin) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/pom.xml]: done

**Review by hailin0 — DISMISSED** [review]

**Review by hailin0 — DISMISSED** [review]
LGTM

**Review by TyrantLucifer — APPROVED** [review]
+1

**Review by EricJoy2048 — APPROVED** [review]
+1

**Pull request merged**